### PR TITLE
add BibTeX file

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: CITATION.bib [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@Misc{montoison-orban-siquiera-nlpmodelsjump-2020,
+  author = {A. Montoison and D. Orban and A. S. Siqueira},
+  title = {{NLPModelsJuMP.jl}: Conversion from JuMP models to NLPModels},
+  month = {August},
+  howpublished = {https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl},
```
Comment (dpo) [inline CITATION.bib:5]: `\url{...}`

File: README.md [hunk 2]
```diff
@@ -1,27 +1,18 @@
...
+NLPModelsJuMP.jl provides conversion from [JuMP.jl](https://github.com/jump-dev/JuMP.jl) / [MathOptInterface.jl](https://github.com/jump-dev/MathOptInterface.jl) models to [NLPModels](https://github.com/JuliaSmoothOptimizers/NLPModels.jl).
 
 **Disclaimer**: NLPModelsJuMP is *not* developed or maintained by the JuMP developers.
 
-See the documentation on NLPModels for the general description of NLPModels. Here, we
+See the [documentation](https://juliasmoothoptimizers.github.io/NLPModels.jl/stable/) on NLPModels for the general description of NLPModels. Here, we
 focus on the use of JuMP to create these.
 
-## NLPModels from a JuMP
+## How to Cite
```
Comment (dpo) [inline README.md:16]: I suggest putting this section at the top.

File: README.md [hunk 3]
```diff
@@ -1,27 +1,18 @@
...
-This package provides nonlinear programming models as implemented by
-[NLPModels](https://github.com/JuliaSmoothOptimizers/NLPModels.jl), through the use of
-[JuMP](https://github.com/JuliaOpt/JuMP.jl).
+NLPModelsJuMP.jl provides conversion from [JuMP.jl](https://github.com/jump-dev/JuMP.jl) / [MathOptInterface.jl](https://github.com/jump-dev/MathOptInterface.jl) models to [NLPModels](https://github.com/JuliaSmoothOptimizers/NLPModels.jl).
 
 **Disclaimer**: NLPModelsJuMP is *not* developed or maintained by the JuMP developers.
 
-See the documentation on NLPModels for the general description of NLPModels. Here, we
+See the [documentation](https://juliasmoothoptimizers.github.io/NLPModels.jl/stable/) on NLPModels for the general description of NLPModels. Here, we
 focus on the use of JuMP to create these.
```
Comment (dpo) [inline README.md:14]: The last sentence seems redundant.

File: README.md (hunk 2 above)
Comment (amontoison) [reply README.md:16]: The README.md is very short, we can't missed it :)
nlpmodelsjump

File: CITATION.bib [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@Misc{montoison-orban-siquiera-nlpmodelsjump-2020,
+  author = {A. Montoison and D. Orban and A. S. Siqueira},
+  title = {{NLPModelsJuMP.jl}: Conversion from {JuMP} models to {NLPModels}},
+  month = {August},
+  howpublished = \url{https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl},
```
Comment (dpo) [inline CITATION.bib:5]: I think it should be `{\url{...}},`

File: README.md (hunk 2 above)
Comment (dpo) [reply README.md:16]: If you put it at the top now, it will always be at the top, even after we add to the README :)

Comment (amontoison) [reply README.md]: That's a good point !

**Comment by amontoison** [comment]
nlpmodelsjump

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**